src/main/java/net/sf/jabref/model/entry/FieldName.java

```diff
@@ -17,6 +17,9 @@
     // author is not set):
     public static final String FIELD_SEPARATOR = "/";
 
+    public static final String ABSTRACT_ALL_FIELD = "all";
+    public static final String ABSTRACT_ALL_TEXT_FIELDS_FIELD = "all_text_fields";
```

Comment:
Can this be named `all-text-fields`. I think dash ("`-`") is more intuitive than underscore ("`_`").